modules/@angular/upgrade/test/upgrade_spec.ts

```
@@ -20,6 +20,65 @@ export function main() {
 
     it('should have angular 1 loaded', () => expect(angular.version.major).toBe(1));
 
+
+
+    describe('errors', () => {
+      let adapter: UpgradeAdapter;
+
+      beforeEach(() => {
+        const ng1Module = angular.module('ng1', []);
+
+        const Ng2Component =
+            Component({selector: 'ng2', template: `<BAD TEMPLATE div></div>`}).Class({
+              constructor: function() {}
+            });
+
+        const Ng2Module = NgModule({declarations: [Ng2Component], imports: [BrowserModule]}).Class({
+          constructor: function() {}
+        });
+
+        adapter = new UpgradeAdapter(Ng2Module);
+      });
+
+
+      it('should be thrown', fakeAsync(() => {
```

Changed the test descriptions so it goes like
"adapter: ng1 to ng2 ... bootstrap errors ... should throw an uncaught error" and
"adapter: ng1 to ng2 ... bootstrap errors ... should properly log to the console and re throw"

I had to use `fakeAsync` to call `flushMicrotasks` and capture the `throw`.